website/source/docs/provisioners/powershell.html.md

```diff
@@ -72,7 +72,20 @@ Optional parameters:
 -   `environment_vars` (array of strings) - An array of key/value pairs to
     inject prior to the execute\_command. The format should be `key=value`.
     Packer injects some environmental variables by default into the
-    environment, as well, which are covered in the section below.
+    environment, as well, which are covered in the section below. If you are
+    using AWS and would like to use the randomly-generated unique
+    If you are running on AWS and would like to access the AWS-generated
+    Administrator password that Packer uses to connect to the instance via
+    WinRM, you can use the template variable `{{.WinRMPassword}}` to set this
+    as an environment variable. For example:
+
+    ```json
+      {
+        "type": "powershell",
+        "environment_vars": "WINRMPASS={{.WinRMPassword}}",
```

Comment:
I think it would be much for convenient to the user to have this just become 

```json
{
    "type": "powershell",
    "inline": ["Write-Host \"Automatically generated aws password is: $Env:PACKER_WINRM_PASSWORD\""]
}
```